src/commands/role.ts

```diff
@@ -179,7 +199,8 @@ class RoleCMD extends Command {
             (role: Role) => {
                 if (
                     botHighestPosition > role.rawPosition &&
-                    role.name.toLowerCase() !== '@everyone'
+                    role.name.toLowerCase() !== '@everyone' &&
+                    role.editable
```

Comment:
We can make a utility function to prevent code duplication.

Reply:
Could you take a look at c2298b4 and check if that's what you had in mind?

Technicaly, `#isAssignableRole` could call itself `#getBotRolePosition` instead of asking for it in a parameter. The performance difference should be insignificant, since it is accessing to static/cached properties.

Currently, the only difference is for `ab role --list` it will call `getBotRolePosition` once, while calling it from `#isAssignableRole` would make it call `N` times... which doesn't matter at all, so just let me know if you prefer the later or find it cleaner. 😄